tensorflow_datasets/video/bair_robot_pushing.py

```diff
@@ -89,17 +90,17 @@ def _split_generators(self, dl_manager):
 
   def _generate_examples(self, filedir):
     logging.info("Reading data from %s.", filedir)
-    files = tf.io.gfile.listdir(filedir)
+    files = compat.v2.io.gfile.listdir(filedir)
     logging.info("%d files found.", len(files))
 
     # For each file
-    for filename in sorted(tf.io.gfile.listdir(filedir)):
+    for filename in sorted(compat.v2.io.gfile.listdir(filedir)):
       filepath = os.path.join(filedir, filename)
 
       # For each video inside the file
       for video_id, example_str in enumerate(
-          tf.compat.v1.io.tf_record_iterator(filepath)):
```

Comment:
The problem is that `tensorflow.compat.v2` was imported as `tf`. And here `tf.compat.v1` has been called which is it points to `tensorflow.compat.v2.compat.v1` which leads to import errors.

Reply:
`tensorflow.compat.v2.compat.v1` is valid too. Make sure you're using a recent version of TF.